packages/basic-crawler/tsconfig.build.json

```diff
@@ -1,7 +1,9 @@
 {
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
-    "outDir": "./dist"
+    "outDir": "./dist",
+    "module": "Node16",
+    "moduleResolution": "Node16"
```

Comment:
this should go to the root tsconfig, or not?

Reply:
It should! But I'll submit a PR to the module to fix their exports too

Reply:
Wrong response, this was supposed to be for the patch review. I've had to add it to the root, next commits will have it